### PR TITLE
Additional changes to dockers

### PR DESCRIPTION
**Pull request by ElsLommelen**
## Description
This PR makes some small changes in the dockers:

1. Environmental variable names are changed
    - to comply with the environmental variables provided by Github during GHA (which ensures we can use these env vars)
    - to comply with the name given to the personal access token in the repo (`PAT`) *
2. In line with issue #61, the branch name is used as an input for variable `protocol_code`, and we verify that only changes are made to the protocol mentioned as variable name.

\* the same name `PAT` is used for the personal access token in the repository `inbo/protocols` that still has to be made.

## Task list

Steps by contributor:

- [x] Add description to this pull request (under "## Description")
- [x] Submit the pull request by clicking 'create pull request'
- [x] Add further commits if needed and push them to GitHub
- [x] Mark the pull request as 'ready for review'

Review steps for the author(s):

- [x] Add reviewers, at least one subject-matter specialist and one administrator
- [ ] Wait for review comments and address them
- [ ] Iterate until reviewer approvals (merging the pull request will be done by an administrator)

To be checked by reviewer:

- [ ] The error message (for clarity)

### CODE REVIEW

File: docker/entrypoint_check.sh [hunk 1]
```diff
@@ -1,18 +1,31 @@
...
+echo '\nCheck if only protocol is updated...\n'
+FOLDERNAME=$(Rscript -e 'protocolhelper::get_path_to_protocol("'$GITHUB_HEAD_REF'")')
+FOLDERNAME_REL=${FOLDERNAME#/home/runner/work/protocolsource/}
+echo 'FOLDERNAME=' $FOLDERNAME
+echo 'FOLDERNAME_REL=' $FOLDERNAME_REL
+CHANGED=$(git diff --name-only main | grep -v NEWS\.md | grep -v \.zenodo\.json| grep -v ^$FOLDERNAME_REL/)
+echo 'CHANGED=' $CHANGED
+
+if [ -n "${CHANGED}" ]; then
+  echo '\nFiles not belonging to the protocol mentioned in the branch name are changed, and these changes will not be passed to the website. Please remove these changes here and start a new branch for changes to other protocols./n\n'
```
Comment (hansvancalster) [inline docker/entrypoint_check.sh:21]:
```suggestion
  echo '\nFiles not belonging to the protocol mentioned in the branch name ("'$GITHUB_HEAD_REF'") are changed, and these changes will not be passed to the website. Please remove these changes from this branch and start a new branch for changes to other protocols./n\n'
```

**Review by hansvancalster — COMMENTED** [review]
Looks good. One suggestion to improve the message.

File: docker/entrypoint_check.sh (hunk 1 above)
Comment (hansvancalster) [reply docker/entrypoint_check.sh:21]: Does `$CHANGED` gives a list of which files were changed? If so, maybe include it at the end of this message?

Comment (ElsLommelen) [reply docker/entrypoint_check.sh]: With respect to the error: would it be a good idea to just replace the whole description `to the protocol mentioned in the branch name` by the branch name?  In case the user has the right protocol code as a branch name, it would be easier to understand.  On the other hand, when he made a mistake in the branch name, only mentioning the branch name and not the description, would make it difficult to understand why the error message is dropped...

Comment (ElsLommelen) [reply docker/entrypoint_check.sh]: > Does `$CHANGED` gives a list of which files were changed? If so, maybe include it at the end of this message?

Yes it does.  I'll add it, do you give your opinion on my previous comment?

Comment (hansvancalster) [reply docker/entrypoint_check.sh]: > With respect to the error: would it be a good idea to just replace the whole description `to the protocol mentioned in the branch name` by the branch name? In case the user has the right protocol code as a branch name, it would be easier to understand. On the other hand, when he made a mistake in the branch name, only mentioning the branch name and not the description, would make it difficult to understand why the error message is dropped...

Maybe reword it as follows:

'\nBranch '$GITHUB_HEAD_REF' contains changes to files which do not belong to protocol '$GITHUB_HEAD_REF', and these changes will not be passed to the website. Please remove these changes from this branch and start a new branch for changes to other protocols./n\n'

Comment (ElsLommelen) [reply docker/entrypoint_check.sh]: Ok, and do I keep the `changes to be removed: xxx` in the end?

File: docker/entrypoint_check.sh [hunk 2]
```diff
@@ -1,22 +1,32 @@
...
+echo '\nCheck if only protocol is updated...\n'
+FOLDERNAME=$(Rscript -e 'protocolhelper::get_path_to_protocol("'$GITHUB_HEAD_REF'")')
+FOLDERNAME_REL=${FOLDERNAME#/home/runner/work/protocolsource/}
+echo 'FOLDERNAME=' $FOLDERNAME
+echo 'FOLDERNAME_REL=' $FOLDERNAME_REL
+CHANGED=$(git diff --name-only main | grep -v NEWS\.md | grep -v \.zenodo\.json| grep -v ^$FOLDERNAME_REL/)
+
+if [ -n "${CHANGED}" ]; then
+  echo '\nBranch '$GITHUB_HEAD_REF' contains changes to files which do not belong to protocol '$GITHUB_HEAD_REF', and these changes will not be passed to the website. Please remove these changes from this branch and start a new branch for changes to other protocols. Changes to be removed:' $CHANGED
+  exit 1
```
Comment (hansvancalster) [inline docker/entrypoint_check.sh:21]: Can you double check the use of single quotation marks? Are nested single quotation marks allowed?
